scripts/configure_dev.sh

```diff
@@ -86,6 +86,7 @@ elif [ "${OS}" = "darwin" ]; then
         install_or_upgrade automake
         install_or_upgrade python3
         install_or_upgrade lnav
+        install_or_upgrade diffutils
```

Comment:
I wasn't able to get `make lint` to pass locally until I installed `diffutils` per these links:
* [lakeFS forum discussion](https://forum.lakefs.io/t/10003060/beginners-question-the-linting-failed-on-https-github-com-tr)
* https://github.com/golangci/golangci-lint/issues/3087